### DISCUSSION TITLE
[Show and Tell] <NFTEarth - non-fungible protocol live on OP Mainnet and Base>

### DISCUSSION THREAD

**Discussion by westonnelson** [ethereum-optimism/developers#174] [Show and tell]
### Demoing: Cross-chain transfer of NFTE LayerZero tokens and discuss how to grow the NFT ecosystem on Layer2s like OP Mainnet and Base. 🔴 
---
## NFTEarth is one of the few NFT protocols with a native protocol token - $NFTE - that is also an Omnichain token built with the LayerZero OFT extension, making it seamlessly interoperable across chains. 

> **- A Mock NFT-infused Dallas Mavericks ticket - used with a QR code for the convenience of fans!**

![Untitled design (6)](https://github.com/ethereum-optimism/developers/assets/29180454/703711ae-fc8f-4115-b11e-22764b304ef4)

**Comment by sbvegan** [comment]
Thanks for sharing!
